client/app/telecom/telephony/billingAccount/billing/repaymentHistory/telecom-telephony-billing-account-billing-repayment-history.controller.js

```diff
@@ -90,11 +66,8 @@ angular.module("managerApp").controller("TelecomTelephonyBillingAccountBillingRe
 
         return self.fetchFile(consumption).then(function (info) {
             $window.location.href = info.url;
-        }).catch(function (error) {
-            Toast.error([$translate.instant("telephony_group_billing_repayment_history_download_error"), (error.data && error.data.message) || ""].join(" "));
-            return $q.reject(error);
-        }).finally(function () {
-            consumption.downloading = false;
+        }, function (error) {
+            return new ToastError([$translate.instant("telephony_group_billing_repayment_history_download_error"), (error.data && error.data.message) || ""].join(" "));
```

Comment:
This is not how `ToastError` works. It takes as arguments an error object AND a translation key, not a string